src/main/java/org/zone/region/flag/player/entitydamage/EntityDamagePlayerListener.java

```diff
@@ -0,0 +1,35 @@
+package org.zone.region.flag.player.entitydamage;
+
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.event.Listener;
+import org.spongepowered.api.event.entity.DamageEntityEvent;
+import org.zone.ZonePlugin;
+import org.zone.region.Zone;
+import org.zone.region.group.Group;
+import org.zone.region.group.key.GroupKeys;
+
+import java.util.Optional;
+
+public class EntityDamagePlayerListener {
+    @Listener
+    public void onEntityDamageEvent(DamageEntityEvent event) {
+
+        if (!(event.entity() instanceof Player player)) {
+            return;
+        }
+
+        Optional<Zone> opZone = ZonePlugin
```

Comment:
Im not seeing a check for damaged by another entity

Reply:
oops